impeller/renderer/backend/gles/buffer_bindings_gles.cc

```diff
@@ -101,6 +103,38 @@ bool BufferBindingsGLES::ReadUniformsBindings(const ProcTableGLES& gl,
   if (!gl.IsProgram(program)) {
     return false;
   }
+  program_handle_ = program;
+  if (gl.GetDescription()->GetGlVersion().IsAtLeast(Version{3, 0, 0})) {
+    return ReadUniformsBindingsV3(gl, program);
+  }
+  return ReadUniformsBindingsV2(gl, program);
+}
+
+bool BufferBindingsGLES::ReadUniformsBindingsV3(const ProcTableGLES& gl,
+                                                GLuint program) {
+  program_handle_ = program;
+  GLint uniform_blocks = 0;
+  gl.GetProgramiv(program, GL_ACTIVE_UNIFORM_BLOCKS, &uniform_blocks);
+  for (GLint i = 0; i < uniform_blocks; i++) {
+    GLint name_length = 0;
+    gl.GetActiveUniformBlockiv(program, i, GL_UNIFORM_BLOCK_NAME_LENGTH,
```

Comment:
Paranoid note: Does the name include the NULL terminator?

Reply:
according to https://registry.khronos.org/OpenGL-Refpages/gl4/html/glGetActiveUniformBlock.xhtml it does